src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -282,12 +312,28 @@ impl DeviceManager {
         config: &VhostUserConfig,
         device_id: String,
     ) -> Result<ArcMutexDevice> {
+        // TODO virtio-scsi
         let mut vhu_blk_config = config.clone();
-        vhu_blk_config.driver_option = self.shared_info.block_driver.clone();
+
+        match vhu_blk_config.driver_option.as_str() {
+            // convert the block driver to kata type
+            VIRTIO_BLOCK_MMIO => {
+                vhu_blk_config.driver_option = KATA_MMIO_BLK_DEV_TYPE.to_string();
+            }
+            VIRTIO_BLOCK_PCI => {
+                vhu_blk_config.driver_option = KATA_BLK_DEV_TYPE.to_string();
+            }
+            _ => {
+                return Err(anyhow!(
+                    "invliad driver type {}",
```

Comment:
Sorry, the "invalid driver type" -> "unsupported driver type" is better.

Reply:
fixed